src/main/app/claims/models/claim.ts

```diff
@@ -411,6 +411,23 @@ export class Claim {
       && this.settlement && this.settlement.isOfferRejected()
   }
 
+  get isSettlementPaymentDateValid (): boolean {
```

Comment:
why is this a getter? i see that you followed above example but i think that's wrong too...if you see most other places in same file...they are all methods. 

I understand we didn't follow same standard everywhere in this file, i wish we did but i think we should do it right going forward..

Reply:
looks like its wrongly called a getter in only 1 other place and you picked that one :)

Reply:
I agree with @KiranVarma02 , imho if something is computed on the fly with arguably complex logic it should be a method - leaving getters for returning properties with little manipulation. Ie. returning an uppercase string out of a property might as well be a getter